test_environments/message_and_event_brokers/kafka/sasl_scram/with_tls/README.md

```diff
@@ -0,0 +1,99 @@
+# Overview
+This is a simple setup of Kafka with SASL_SCRAM (SHA-256 algorith) authentication and TLS encryption. 
```

Comment:
I think this is a copy-paste from one of the folders below (because this README should mention both 256 and 512 algo)

Reply:
Fixed.